pkgs/top-level/python-packages.nix

```diff
@@ -152,6 +152,32 @@ in {
     };
   };
 
+  ansicolor = buildPythonPackage rec {
+    name = "ansicolor-${version}";
+    version = "0.2.4";
+
+    src = pkgs.fetchurl{
```

Comment:
@rasendubi: I changed to pypi. Interestingly, there are no tests in the pypi package of ansicolor. Therefore I bypassed the test problem. I've verified that the library still works by running its demos.